packages/openid4vc/src/openid4vc-verifier/__tests__/openid4vc-verifier.test.ts

```diff
@@ -113,5 +113,44 @@ describe('OpenId4VcVerifier', () => {
       expect(jwt.payload.iss).toEqual(verifier.did)
       expect(jwt.payload.sub).toEqual(verifier.did)
     })
+
+    it('check openid proof request format (entity id)', async () => {
+      const openIdVerifier = await verifier.agent.modules.openId4VcVerifier.createVerifier()
+      const { authorizationRequest, verificationSession } =
+        await verifier.agent.modules.openId4VcVerifier.createAuthorizationRequest({
+          requestSigner: {
+            method: 'openid-federation',
+            clientId: 'http://localhost:3001/verifier',
```

Comment:
Can't we infer the client id based on the hosted url? The federation url must be the verifier url, so we can jsut construct that? no need to pass the client id? just passing `openid-federation` method should be enough